delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/indexing/IndexingState.scala

```diff
@@ -0,0 +1,10 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.indexing
+
+import ch.epfl.bluebrain.nexus.sourcing.projections.stream.StreamSupervisor
+
+sealed private[indexing] trait IndexingState extends Product with Serializable
+
+private[indexing] object IndexingState {
+  final case object Initial                                         extends IndexingState
+  final case class Current(rev: Long, supervisor: StreamSupervisor) extends IndexingState
```

Comment:
It may be another PR but it would be nice to know when it has been started for example

Reply:
This information is already present in the progress offset, which as per your request is handled separately in a distributed data cache

Reply:
I meant when the current StreamSupervisor started, this part is not in progress offset.

Reply:
We don't care about when it started, we care about the progress.

If we ever care about when it started, I will add it